changelogs/fragments/1176-ssm-connection-plugin-region-fix.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
```

```suggestion
bugfixes:
```

I wonder if we should flag this as a bugfix?